release.nix

```diff
@@ -22,6 +22,9 @@ let
   makeJobs = cluster: with daedalusPkgs { inherit cluster; }; {
     inherit daedalus;
     installer = wrappedBundle newBundle pkgs cluster daedalus-bridge.version;
+    devops-970 = {
```

Comment:
should probably be renamed to something like windowsInstaller (and the file imported to windows-installer).

Reply:
i plan to merge most of those expressions into either `default.nix` or `release.nix` and get rid of the new file entirely